starfish/imagestack/imagestack.py

```diff
@@ -386,6 +387,28 @@ def set_slice(
 
         self._data.values[slice_list] = data
 
+    def show_stack_napari(self, indices: Mapping[Indices, Union[int, slice]]):
```

Comment:
This is awesome. Is it possible to have napari label the sliders? I find that part of it a bit unintuitive. 

If it's _not_ possible, we should document the order that they appear on the GUI from top to bottom. Based on the MERFISH example, seems like it's the same as the `reordered_array`?